io/disk/disktest.py

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env python
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+# Copyright: 2016 IBM
+# Author: Nosheen Pathan <nopathan@linux.vnet.ibm.com>
+# Based on code by
+# Author: Martin Bligh (mbligh@google.com)
+# Copyright: 2007 Google, Inc.
+
+import os
+import shutil
+
+from avocado import Test
+from avocado import main
+from avocado.utils import memory
+from avocado.utils import disk
+from avocado.utils import process
+from avocado.utils import build
+from avocado.utils.software_manager import SoftwareManager
+
+
+class Disktest(Test):
+
+    """
+    Avocado module for disktest.
+    Pattern test of the disk, using unique signatures for each block and each
+    iteration of the test. Designed to check for data corruption issues in the
+    disk and disk controller.
+    It writes 50MB/s of 500KB size ops.
+    """
+    version = 2
```

Comment:
This value is not used anywhere in avocado (autotest allowed such API). So it should be removed.